src/ansi-c/library_check.sh

```diff
@@ -7,6 +7,7 @@ for f in "$@"; do
     echo "Checking ${f}"
     cp "${f}" __libcheck.c
     perl -p -i -e 's/(__builtin_[^v])/s$1/' __libcheck.c
+    perl -p -i -e 's/s(__builtin_unreachable)/$1/' __libcheck.c
```

Comment:
Couldn't you just do this for `__CPROVER_assume(0)` instead?

Reply:
That would be a valid hack, but then it's only `__CPROVER_assume(0)` that actually makes the code thereafter unreachable, so it's a bit strange perhaps?